examples/text-generation/run_generation.py

```diff
@@ -320,6 +320,9 @@ def setup_parser(parser):
         action="store_true",
         help="Run the inference with dataset for specified --n_iterations(default:5)",
     )
+    parser.add_argument(
```

Comment:
--sdp_on_bf16 is not added to README. should I update all of the model in run_generation also?

Reply:
not sure for all models. thats a libin qtn